### PR TITLE
fix(apigatewayv2): `WebSocketAwsIntegration` ignores `requestParameters`

### PR DESCRIPTION
**Pull request by FayezX**
https://github.com/aws/aws-cdk/pull/28718 missed adding `requestParameters` to the `WebSocketAwsIntegration`.

### CODE REVIEW

**Review by kaizencc — DISMISSED** [review]
Hi @FayezX, thanks for the PR! Can we get a unit test to demonstrate that things are getting piped through correctly?

**Comment by GavinZZ** [comment]
Thanks for making this fix. I apologize that I missed the property on my last PR. Would be happy to approve once a small unit test is added.

File: packages/@aws-cdk-testing/framework-integ/test/aws-apigatewayv2-integrations/test/websocket/integ.aws.ts [hunk 1]
```diff
@@ -42,6 +42,9 @@ webSocketApi.addRoute('$connect', {
     integrationUri: `arn:aws:apigateway:${stack.region}:dynamodb:action/PutItem`,
     integrationMethod: HttpMethod.POST,
     credentialsRole: apiRole,
+    requestParameters: {
```
Comment (GavinZZ) [inline packages/@aws-cdk-testing/framework-integ/test/aws-apigatewayv2-integrations/test/websocket/integ.aws.ts:45]: This won't be enough unfortunately, you'll need to build this change and run `yarn integ <path-to-this-file> --update-on-failed` to update the snapshots as well.

**Comment by FayezX** [comment]
Hi @kaizencc @GavinZZ can you please take a look to see if this is good or am I missing anything else? Not that familiar with the code base / test strategy.

File: packages/@aws-cdk-testing/framework-integ/test/aws-apigatewayv2-integrations/test/websocket/integ.aws.ts (hunk 1 above)
Comment (FayezX) [reply packages/@aws-cdk-testing/framework-integ/test/aws-apigatewayv2-integrations/test/websocket/integ.aws.ts:45]: Hey Gavin, I think i will need assistance on this. Unfortunately I don't know this library very well. Can you help out please?

Comment (GavinZZ) [reply packages/@aws-cdk-testing/framework-integ/test/aws-apigatewayv2-integrations/test/websocket/integ.aws.ts:45]:
```
npx lerna run build --scope=@aws-cdk-testing/framework-integ
cd packages/@aws-cdk-testing/framework-integ
yarn integ test/aws-apigatewayv2-integrations/test/websocket/integ.aws.js --update-on-failed
```

Comment (FayezX) [reply packages/@aws-cdk-testing/framework-integ/test/aws-apigatewayv2-integrations/test/websocket/integ.aws.ts:45]: I am having some typescript compilation issues and the commands fail for me. Please feel free to add to the pr as you see fit.

Comment (GavinZZ) [reply packages/@aws-cdk-testing/framework-integ/test/aws-apigatewayv2-integrations/test/websocket/integ.aws.ts:45]: It seems that the command itself is not working. Try `yarn integ --update-on-failed` instead and it should work for you.

Comment (FayezX) [reply packages/@aws-cdk-testing/framework-integ/test/aws-apigatewayv2-integrations/test/websocket/integ.aws.ts:45]: running the command `yarn integ --update-on-failed` gives me yarn integ --update-on-failed Error: Cannot find module './integ-runner.js'.  The command seems to be trying to run /packages/@aws-cdk/integ-runner/bin/integ-runner but cannot find the ./integ-runner.js file and fails.

**Comment by FayezX** [comment]
Looks like there is another Pr which fixes this. Please take a look here https://github.com/aws/aws-cdk/pull/28905

**Pull request closed without merging**